<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alt::TransactionTabWidget</class>
 <widget class="QWidget" name="alt::TransactionTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="componentsTab">
      <attribute name="title">
       <string>Components</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="componentRemoveLayout">
         <item>
          <widget class="QLabel" name="removeComponentLabel">
           <property name="text">
            <string>For removal (%1):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="removeComponents"/>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="componentInstallLayout">
         <item>
          <widget class="QLabel" name="installComponentLabel">
           <property name="text">
            <string>For installation (%1):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="installComponents"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="packagesTab">
      <attribute name="title">
       <string>Packages</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="installLabel">
           <property name="text">
            <string>For installation (%1):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="installPackages"/>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="removeLabel">
           <property name="text">
            <string>For removal (%1):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="removePackages"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
